src/Stratis.Bitcoin/Connection/ConnectionManagerController.cs

```diff
@@ -45,12 +54,46 @@ public bool AddNode(string endpointStr, string command)
             return true;
         }
 
+        /// <summary>
+        /// Adds a node to the connection manager.
+        /// API implementation of RPC call.
+        /// </summary>
+        /// <param name="request">A <see cref="AddNodeRequestModel"/> formatted request containing an endpoint and command.</param>
+        /// <returns>Json formatted <c>True</c> indicating success. Returns <see cref="IActionResult"/> formatted exception if fails.</returns>
+        /// <exception cref="ArgumentNullException">Thrown if either request.Endpoint or request.Command are null or empty.</exception>
+        /// <exception cref="ArgumentException">Thrown if request.Command is invalid/not supported.</exception>
+        [Route("addnode")]
+        [HttpGet]
+        public IActionResult AddNode(AddNodeRequestModel request)
```

Comment:
please use 
`public IActionResult AddNodeAPI(string endpoint, string command)`

instead of adding a new request model.